ui/raidboss/data/03-hw/dungeon/sohm_al_hard.js

```diff
@@ -188,32 +188,20 @@
       'locale': 'de',
       'replaceSync': {
         'The Wound': 'Wunde',
-        'The Fever': 'Das Fieber',
         'The Lava Tube': 'Lavagrotte',
-        'The Scorpion\'s Tail': 'Schwanzskorpion',
         'The Leightonward': 'Hortigolem',
-        '(?<! )Spore Sac': 'Sporensack',
-        'Small Spore Sac': 'klein[a] Sporensack',
+        'Small Spore Sac': 'klein(?:e|er|es|en) Sporensack',
```

Comment:
"remove not used timeline translations" ... but also make some modifications too!